### PR TITLE
ExternalLicenseProvider made thread safe

### PR DESCRIPTION
**Pull request by michalszynkiewicz**

### CODE REVIEW

File: src/main/java/me/snowdrop/licenses/sanitiser/provider/ExternalLicenseProvider.java [hunk 1]
```diff
@@ -20,9 +20,9 @@
 import com.fasterxml.jackson.databind.type.TypeFactory;
 import me.snowdrop.licenses.LicensesGeneratorException;
 import me.snowdrop.licenses.xml.LicenseElement;
+import org.jboss.resteasy.client.jaxrs.ResteasyClient;
+import org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder;
 
-import javax.ws.rs.client.ClientBuilder;
```
Comment (gytis) [inline src/main/java/me/snowdrop/licenses/sanitiser/provider/ExternalLicenseProvider.java]: What's the issue with using javax.ws.rs API?

Comment (michalszynkiewicz) [reply src/main/java/me/snowdrop/licenses/sanitiser/provider/ExternalLicenseProvider.java]: RESTEasy implementation of the JAX-RS client turned out not to be thread safe.
Some details are here: https://stackoverflow.com/questions/24700798/is-jax-rs-client-thread-safe#27427911

Comment (michalszynkiewicz) [reply src/main/java/me/snowdrop/licenses/sanitiser/provider/ExternalLicenseProvider.java]: RESTEasy implementation of the JAX-RS client turned out not to be thread safe.
Some details are here: https://stackoverflow.com/questions/24700798/is-jax-rs-client-thread-safe#27427911

Comment (gytis) [reply src/main/java/me/snowdrop/licenses/sanitiser/provider/ExternalLicenseProvider.java]: Fair enough, but everything is running on a single thread so thread-safety shouldn't be an issue.

Comment (michalszynkiewicz) [reply src/main/java/me/snowdrop/licenses/sanitiser/provider/ExternalLicenseProvider.java]: I don't think it is:
https://github.com/snowdrop/licenses-generator/blob/master/src/main/java/me/snowdrop/licenses/LicenseSummaryFactory.java#L43

**Pull request merged**